xivModdingFramework/Exd/FileTypes/Ex.cs

```diff
@@ -169,7 +169,19 @@ private void ReadExHeader(XivEx exFile)
                 var exdFolderHash = HashGenerator.GetHash("exd");
                 var exdFileHash = HashGenerator.GetHash(exdFile);
 
-                exdOffsetList.Add(index.GetDataOffset(exdFolderHash, exdFileHash, XivDataFile._0A_Exd));
+                //esrinzou for china ffxiv
```

Comment:
This seems to be a temporary workaround.

This is unnecessary as the language code should be correct upon starting the application, there will be a check for system language at startup and/or a selection screen.

Reply:
Because the internal file distribution of the game with CHS is different from the description of your original code.

Reply:
This is really a temporary solution, but it should be the simplest solution. I can't find the rule of CHS file distribution. It's too troublesome to test one by one.